build.gradle

```diff
@@ -155,7 +159,7 @@ publishing {
 
   repositories {
     maven {
-      if (repoUrl) {
+      if (project.hasProperty('repoUrl')) {
```

Comment:
I had to do this, as even the `test` task was failing saying that repoUrl was not a valid project property. Can you please check this?